scripts/codegen.js

```diff
@@ -15,13 +15,13 @@ telescope({
     "protos/neutron",
     "protos/slinky",
     "protos/wasmd",
-    "protos/interchain-security",
   ],
   outPath: outPath,
   options: {
     logLevel: 0,
     useSDKTypes: false,
     tsDisable: {
+      disableAll: true,
```

Comment:
why does this do?

Reply:
are we okay with that and why?

Reply:
no other way, it's just the way autogeneration works kinda faulty, some other libs disable it as well. I can recheck if everything is the case as i said.

Reply:
hey did you make another check?